docs/libcudacxx/ptx/instructions/generated/st_async.rst

```diff
@@ -20,8 +7,7 @@ st.async.weak.shared::cluster.mbarrier::complete_tx::bytes.b32
    template <typename Type>
    __device__ static inline void st_async(
      Type* addr,
-     const Type& value,
-     uint64_t* remote_bar);
+     const Type& value,     uint64_t* remote_bar);
```

Comment:
Splitting the files was all done manually, so I can align better with what the generator outputs. This should also make the review easier, because there should be no changes to the content. I will then overwrite all the generated files in a separate PR. 

So this change is a mistake on my side.